framework/vkdf-buffer.cpp

```diff
@@ -44,6 +44,12 @@ vkdf_create_buffer(VkdfContext *ctx,
    return buffer;
 }
 
+static inline uint64_t
+align64(uint64_t value, unsigned alignment)
+{
+   return (value + alignment - 1) & ~((uint64_t)alignment - 1);
```

Comment:
This only works for values of alignment that are powers of 2, so an assert would be useful (the assert could check that only one bit in alignment is set)